src/PockyBot.NET/Constants/Actions.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace PockyBot.NET.Constants
+{
+    class Actions
+    {
+        public const string Get = "get";
+        public const string Set = "set";
+        public const string Delete = "delete";
```

Comment:
I'm confused about this. Does it not already exist? I have it on my branch, and intellisense says it was created by Laura.

Reply:
Side note, why not declare the class `internal static`?

Reply:
It doesn't exist on master yet. I'll handle conflicts

Reply:
ConfigActions exists on master?